content/en/community/contributing/disclosing-vulnerabilities.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
Great call to make this a separate page!

Reply:
thanks!